app/src/main/cpp/BrowserWorld.cpp

```diff
@@ -973,6 +973,7 @@ BrowserWorld::InitializeJava(JNIEnv* aEnv, jobject& aActivity, jobject& aAssetMa
     m.controllers->InitializeBeam();
     m.controllers->SetPointerColor(vrb::Color(VRBrowser::GetPointerColor()));
     m.rootController->AddNode(m.controllers->GetRoot());
+    VRBrowser::OnControllersAvailable();
```

Comment:
Hmm this does not guarantee that the device has physical controllers. In the case of OpenXR this callback is executed whenever the device delegate has created the OpenXRInput object which does happen with or without physical controllers.

Reply:
I see; am I wrong assuming that a non-null ActiveInputMapping indicates the presence of physical controllers ? In that case, we can use the OnControllersReady callback, which is only called when there is an active input mapping,  I know that this could happen as well for devices using the hand tracking input profile, but in that case we could check whether there is more than one input mapping. 

What do you think ?

Reply:
Yes I think we could assume that a non-null active input mapping (unless it's the hand interaction profile) means having physical controllers.